src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -496,15 +495,20 @@ it back */
 	height: 900px;
 }
 
+:deep(.modal-wrapper .modal-container) {
+			height: max-content;
+}
+
 .navigation {
+	position: sticky;
+    bottom: -1px;
```

Comment:
Check indentation here as well. You're using spaces instead of tabs?

Reply:
no? and I also applied a lint fix as well